README.md

```diff
@@ -29,8 +29,6 @@ Master in Geomatics @ Delft University of Technology
 Contact: mhscience@gmail.com  
 
 #### Contributors
```

Comment:
I can not delete people that contributed to this repository during the master thesis but I will modify the Readme to add the paper's contributor,, therefore the contributor's section will highlight who worked on what.

- Contributors 
 -- Paper XYZ (date)
-- Master Thesis (date)

Reply:
I see. I propose to make a new repository from scratch, on my account, in which I'll also add an href to this one.

Reply:
Ok Perfect!